MapboxNavigation.podspec

```diff
@@ -44,7 +44,7 @@ Pod::Spec.new do |s|
   s.module_name = "MapboxNavigation"
 
   s.dependency "MapboxCoreNavigation", "#{s.version.to_s}"
-  s.dependency "MapboxMaps", "10.3.0-beta.1"
+  s.dependency "MapboxMaps", "10.3.0-rc.1"
```

Comment:
We should either use `~>` here or even `>=` and `<` to match `from:` in SPM or use `exact` in SPM.

Reply:
let's go with exact in spm